### PR TITLE
Pract.04 : Add 09-11 | Stefka Lyaskalieva

### PR DESCRIPTION
**Pull request by stefka04**
Stefka Lyaskalieva
2 4MI0600158

### CODE REVIEW

File: SI/Sem.04/Pract.04/Task09.cpp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+using namespace std;
+int calculate(int a, int b, char action)
+{    
+        switch (action)
+        {
+        case '-': return a - b; break;
+        case '*': return a * b; break;
+        case '/': return a / b; break;
+        case '%': return a % b; break;
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task09.cpp:10]: What if b is equal to 0

File: SI/Sem.04/Pract.04/Task10.cpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+using namespace std;
+int binarySystem(int num)
+{
+    int binaryNum = 0; 
+    int i = 1;
+    while (num > 0)
+    {
+       binaryNum = binaryNum + (num % 2)*i;
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task10.cpp:9]: You can use +=

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]

File: SI/Sem.04/Pract.04/Task09.cpp [hunk 3]
```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+using namespace std;
+int calculate(int a, int b, char action)
+{    
+        if (b == 0 && action == '/')
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task09.cpp:5]: You are still not handling the case when the action is %

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]

**Comment by Justsvetoslavov** [comment]
Closing due to merged solutions.

**Pull request closed without merging**